src/test/mir-opt/issue-73223.rs

```diff
@@ -1,3 +1,5 @@
+// compile-flags: -Zmir-opt-level=2 -Zunsound-mir-opts
```

Comment:
The diffs of this file change with more than just line numbers, what's the difference to before?

Reply:
I added a commit message explaining this. Basically, whoever implemented `-Zunsound-mir-opts` forgot to add the flag to MIR opt tests of unsound passes, so these no longer test anything: the pass is just a no-op. Now the pass doesn't even run without `-Zunsound-mir-opts`, so there's no MIR dump, meaning I have to fix this now.

Reply:
ok, makes sense, thanks!